src/Codeception/Configuration.php

```diff
@@ -670,7 +670,7 @@ public static function mergeConfigs($a1, $a2)
 
         // for sequential arrays
         if (isset($a1[0], $a2[0])) {
-            return array_merge_recursive($a2, $a1);
+            return array_merge(array_unique(array_merge_recursive($a2, $a1), SORT_REGULAR));
```

Comment:
Why are you using `array_merge` here?
I've never seen it used with a single argument.